### ISSUE TITLE
UI: Add settings Panel

### ISSUE THREAD

**Issue by edouardparis** [re-vault/revault-gui#21]
Currently the settings button on the sidebar (view/sidebar.rs) is sending a message::Install, 
It should send message::Menu(Settings)

- [ ] modify revaultd config in order to include every missing sections, check revaultd repository
- [ ] create Menu::Settings and modify sidebar view to send the message::Menu(Menu::Settings)
- [ ] create a new `State` call SettingsState, (find inspiration with HistoryState) with a view SettingsView
- [ ] Handle the message and load the SettingsState in the app

The state should handle the app Config and keep it loaded.
First we will only display the config and will not modify it.

For the view: check the config file and for each section, implement a new `card` for each of the following sections:

* bitcoind config
* coordinator
* stakeholders (xpubs)
* managers (xpubs)
* cosigner (xpubs)
* emergency (emergency address)

Inspiration for the design:
![Screenshot_2021-02-11 Figma](https://user-images.githubusercontent.com/6933020/107618944-a79af300-6c52-11eb-9ea2-2266c5bec4bb.png)

**Comment by danielabrozzoni** [comment]
Remaining after #57:
- [ ] Modify the configuration
- [ ] Open/fold the cards

**Comment by edouardparis** [comment]
feedbacks from @JSwambo 
```
Settings
 - has nothing to change. Maybe this could be "configuration details" or "revault parameters" or something? also under bitcoind it says "select the network..." which is confusing. It's already selected, and what am I supposed to do when I read that? there's nothing to select. 
- Why are there cosigners in Manager config, and cosigner keys at the end? why not have a cosigners section? 
- Will stakeholders have names? photos? 
- Will managers have names? photos? 
- My xpub is shown twice... is that necessary? 
- My info is in "Manager Config"... not "My parameters" or something that makes it clear it's mine. Also... "config" is not descriptive to a lay person. "Configuration" or "Parameters" or "Info" might be better. 
```

**Comment by danielabrozzoni** [comment]
![Revault GUI_008](https://user-images.githubusercontent.com/25042473/114439638-7fcff900-9bc9-11eb-9bac-6408237f12fe.png)

"xpub" title for manager info is covered by the actual xpub :(

**Comment by danielabrozzoni** [comment]
Also, longer fields are trimmed, see https://github.com/revault/revault-gui/pull/119#issuecomment-836709428

**Comment by darosior** [comment]
Small note just to make sure we are on the same page: most of the fields that are presents in the current "settings" panel should not be set-able and therefore not present (eg the descriptors).

**Comment by danielabrozzoni** [comment]
Do you think they should be in a different view for non set-able info or we shouldn't display them altogether?

**Comment by darosior** [comment]
I don't think we should display them altogether. Or at least, not in this way

**Comment by edouardparis** [comment]
will be closed by #322
